RPIPs/RPIP-49.md

```diff
@@ -1,68 +1,127 @@
-
-## Topics not being addressed in this rework
-- DAO portion of inflation (30% of the current 5% per annum)
-  - There were some discussions about whether we are better off continuing to fund things with RPL or ETH as we get to maturity. This issue isn't pressing. For now we'll continue using RPL inflation. We can discuss comfortably later and vote as desired.
-- MEV penalty improvements - this _is_ important, but the numbers we've seen previously suggest MEV theft has not been a pressing problem.
-- rETH restitution from underperforming Node Operators - this is moderate priority, significant complexity, and has some different variants possible (see [ArtDemocrat's original proposal](https://dao.rocketpool.net/t/rapid-research-incubator-submission-reth-protection-through-rpl-rerouting-deflation/2599) or [Val's tweak](https://dao.rocketpool.net/t/rapid-research-incubator-submission-reth-protection-through-rpl-rerouting-deflation/2599)). While I wish to see it, I don't think it's worth risking timeline on the [components above](#components-currently-being-advocated-for).
-- Right now voter share is split up amongst potential voters based on amount of RPL eligible to vote. We could do something based on active governance or actual vote power instead of count of RPL?
-  - Possible thought: require having a delegate set for eligibility
-  - Possible thought: setting a delegate expires after a time period (eg, 1 year)
-  - Possible thought: if a delegate has x% of vote power, prevent new delegations to them (revert with error)
-  - Simplicity is being favored more than these; however, this may be something to revisit, depending on the actual practice of governance observed
+## Abstract
+This informational RPIP provides an introduction and overview to the 2024 community tokenomics rework, its likely contents, and its current status. It exists to explain the overall tokenomics proposal as it's currently envisioned by contributing community members. This RPIP lists and briefly describes the rework's components, and links out to the RPIPs that specify those components in greater detail.
+
+The tokenomics rework will likely be split into two protocol upgrades, Saturn I and Saturn II. This RPIP describes the components that are expected to be included within each.
+
+This Informational RPIP and the tokenomics rework represent the best efforts of the Rocket Pool community contributors involved. Information within this RPIP should not be considered official word from the Rocket Pool core development team. 
+
+## Contents
+
+### [RPIP-43: Megapools](RPIP-43.md)
+
+A Megapool is a single contract that can be used as an ethereum withdrawal address for multiple validators.
+* This allows for much more gas-efficient usage of the Rocket Pool protocol for Node Operators.
+* This allows for the application of node-level penalities. 
+* Importantly, this Megapool specification allows Node Operators to create ETH-only validators. 
```

Comment:
Importantly, this RPIP specifies that Node Operation no longer requires RPL staking (up to and including ETH-only node operation)